forJIS7/src/main/java/JavaHomework/Lectures_9/Interfaces/Animal/Animals.java

```diff
@@ -0,0 +1,53 @@
+package JavaHomework.Lectures_9.Interfaces.Animal;
+
+import java.util.Objects;
+
+public class Animals {
+
+    private String animalsName;
```

Comment:
just String name; make it more abstract!

Reply:
fixed